src/mon/MonOpRequest.h

```diff
@@ -86,7 +86,8 @@ struct MonOpRequest : public TrackedOp {
   op_type_t op_type;
 
   MonOpRequest(Message *req, OpTracker *tracker) :
-    TrackedOp(tracker, req->get_recv_stamp()),
+    TrackedOp(tracker,
+      req->get_recv_stamp() ? req->get_recv_stamp() : ceph_clock_now()),
```

Comment:
you are actually using the `operator double` as `operator bool`. could you instead use `is_zero()` instead? and wrap the line at 80 chars.